resources/logging/charts/loki/templates/kyma-additions/grafana-datasource.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v1
```

Comment:
We also make this and the one in tracing configurable just like in prometheus-istio.